test/models/requests/test.json

```diff
@@ -0,0 +1 @@
+{ "errorDetail": { "timestamp": "2019-08-16T18:15:41Z", "correlationId": "3jh1f6b3-f8b1-4f3c-973a-05b4720e", "errorCode": "400", "errorMessage": "Bad request received", "source": "Backend", "sourceFaultDetail": { "detail": [ "Invalid value supplied for field statementRequestId: 32 | Invalid value supplied for field failureReason: NoDocs " ] } }}
```

Comment:
Here

Reply:
Done, file is not needed. hence removed.